### ISSUE TITLE
Is modifying this exception message allowed for clarity? 

### ISSUE THREAD

**Issue by vicelyav** [nus-cs2103-AY2425S1/forum#740]
Hi, I want to ask, we have a command where its parameters can be one of the following combinations:
j/ t/  or t/ or j/ (excluding those accepted)
<img width="758" alt="Screenshot 2024-11-12 at 12 32 00 AM" src="https://github.com/user-attachments/assets/1ddeef92-5e0c-4f18-8046-56221c097b13">
Right now, the exception message is as above. We think that it might be misleading, as it seems that the user must input all consecutively, instead of being individual combinations inputted separately. 
Additionally, for the last line, the 'excluding accepted' is misleading, as accepted is not a JobCode but a tag. Is it alright to modify this message?

**Comment by damithc** [comment]
@vicelyav Yes, you can revise this error message to avoid misleading the reader.
